languages/python/oso/polar/polar.py

```diff
@@ -64,6 +64,7 @@ class Polar:
     def __init__(self, classes=CLASSES):
         self.ffi_polar = FfiPolar()
         self.host = Host(self.ffi_polar)
+        self.ffi_polar.host = self.host
```

Comment:
This could be something like `self.ffi_polar.set_message_enricher(callable)` if you want to take my suggestion from above.